source/_integrations/homeseer.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: HomeSeer
+description: Connect and control your HomeSeer Z-Wave devices using the Z-Wave integration
+ha_release: '2023.4'
+ha_iot_class: Local Push
+ha_category:
+  - Light
+  - Plug
+  - Sensor
+  - Switch
+  - Update
```

Comment:
🤔
I thought they would, actually... I'll clarify with Dominic

Reply:
So, there was a small misinterpretation in terms of urgency and remaining work to get this up and running! Having you point it out @MartinHjelmare surfaced it, thus: Thanks a lot 🙏!

We do have the commitment from HomeSeer to offer firmware updates via Z-Wave JS, it's just not yet ready. Current plan is end of next week, though that's a pretty rough estimate given they only read the docs and not started any implementation...

I would propose to rely on the promise given and list "Update" already, mainly because if not IMHO I would need to also adjust the text.

Any strong opinion @MartinHjelmare @frenck